src/main/java/net/dries007/tfc/common/entities/GenderedRenderAnimal.java

```diff
@@ -6,9 +6,19 @@
 
 package net.dries007.tfc.common.entities;
 
+import net.minecraft.resources.ResourceLocation;
+
+import net.dries007.tfc.util.Helpers;
+
 public interface GenderedRenderAnimal
 {
     boolean displayMaleCharacteristics();
 
     boolean displayFemaleCharacteristics();
+
+    public static <T> ResourceLocation getGenderedTexture(GenderedRenderAnimal animal, String name)
```

Comment:
this is duplicated in WildAnimal? also this should probably be next to the `animalTexture` method in `Helpers`.

Reply:
for esoteric performance reasons, can this be made into

```
final ResourceLocation male = Helpers.animalTexture(name + "_male")
final ResourceLocation female = Helpers.animalTexture(name + "_female")
return animal -> animal.displayMaleCharacteristics() ? name + "_male" : name + "_female";
```

Reply:
Changed to

`final ResourceLocation male = Helpers.animalTexture(name + "_male");
final ResourceLocation female = Helpers.animalTexture(name + "_female");
return animal.displayMaleCharacteristics() ? male : female;`